extensions/binding/org.eclipse.smarthome.binding.sonos/README.md

```diff
@@ -22,7 +22,12 @@ The Sonos devices are discovered through UPnP in the local network and all devic
 
 ## Binding Configuration
 
-The binding does not require any special configuration
+The binding has the following configuration options, which can be set for "binding:sonos":
+
+| Parameter | Name    | Description  | Required |
+|-----------------|------------------------|--------------|------------ |
+| opmlUrl | OPML Service URL | URL for the OPML/tunein.com service | no |
+| callbackUrl | Callback URL | URL to use for playing notification sounds, e.g. http://192.168.0.2:8080 | no |
```

Comment:
I've been having a think about the callbackUrl for audiosinks.  What if I have two similar devices/Things and I want to specify on-the-fly which device plays the audio? What are you thoughts on having the callbackUrl as a Thing config rather than a binding config?